tests/keras/applications/applications_test.py

```diff
@@ -174,6 +174,34 @@ def test_inceptionv3_variable_input_channels():
 
 
 @keras_test
+def test_inceptionresnetv1_notop():
+    def target(queue):
+        model = applications.InceptionResNetV1(weights=None, include_top=False)
+        queue.put(model.output_shape)
```

Comment:
Is the workaround with multiprocessing really necessary for this test? I suppose the Inception-ResNet V1 model should be of similar size to Inception V3. Can you verify it (on CNTK backend)?